riscv/processor.cc

```diff
@@ -248,6 +248,11 @@ void processor_t::parse_isa_string(const char* str)
 
   while (*p) {
     if (islower(*p)) {
+      switch (*p) {
+        case 'q': max_isa |= 1L << ('d' - 'a');
+        case 'd': max_isa |= 1L << ('f' - 'a');
+        // case 'f': extension_table[EXT_ZICSR] = true;
```

Comment:
Please don't include the commented-out code.

Note that Spike _necessarily_ has Zicsr, because Zicsr is implied by the privileged architecture.  We don't need to call it out specially.  I suppose we could accept the string and ignore it, but we shouldn't require the string.